### PR TITLE
Add thank you Page

### LINKED ISSUE #29: Developing Slack Inviter UI in Jade
## Description
As a contributor,
I need to develop and polish the Slack Inviter UI in Pug.

### Update [Required]
- [ ] The first page with the Systers OSS picture in the background, and the Authenticate with Github Button
- [ ] The second page with the Details of the User and the Terms and Conditions
- [ ] The third "Thank You" or "Error" Page.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
8 hours 

@ancyp @bethanyr @Janiceilene Can you please assign me this issue to work upon?

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Developed the Slack Inviter Thank You page in Jade.
The user navigates to the page, after getting the email invite sent to them.

Fixes #29 

Time Taken:
4 hours

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
screen shot 2018-08-08 at 7 25 49 pm
Tested the UI locally.
Tested the NodeJS Routing Part by replacing the navigation from the "Authenticate using github" button directing towards the Thank You page.(Since on clicking on the Authenticate using Github button it takes me to the callback URL which is the Heroku deployment of the app and thus any local changes cannot be checked any further).

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules
 

@ancyp @bethanyr @Janiceilene Can you please review the PR? Thanks! :)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -39,6 +39,12 @@ app.use("/", index);
 app.use("/users", users);
 app.use("/auth", auth);
 
+app.get('/success', function(req, res){
+  res.render('thankyou', {
```
Comment (bethanyr) [inline app.js]: We must not have codacy running in this repo, can you use double quotes to be consistent?

**Comment by bethanyr** [comment]
The rest looks good to me, just one comment

File: app.js (hunk 1 above)
Comment (BaaniLeen) [reply app.js]: Sure. Done! :)

**Comment by BaaniLeen** [comment]
Sure @bethanyr ! Done! :D

**Comment by Janiceilene** [comment]
Maybe take off the smiley face :) Other than that it looks good!

**Comment by BaaniLeen** [comment]
Done! @Janiceilene :)

**Pull request merged**